spec/src/main/asciidoc/converters.asciidoc

```diff
@@ -76,18 +76,18 @@ If an Implicit Converter cannot convert a value, a `java.lang.IllegalArgumentExc
 For the built-in converters and custom converters, the corresponding Array converters are provided by default.
 The delimiter for the config value is ",".
 The escape character is "\".
-e.g. With this config `myPets=dog,cat,dog\,cat`, the values as an array will be `{"dog", "cat", "dog,cat"}`.
+e.g. With this config `myPets=dog,cat,dog\,cat`, the values as an array will be `{"dog", "cat", "dog", "cat"}`.
 
 ==== Programmatic lookup
 
 Array as a class type is supported in the programmatic lookup.
 
 [source, java]
 ----
-String[] myPets = ConfigProvider.getValue("myPet", String[].class);
+String[] myPets = ConfigProvider.getConfig().getValue("myPet", String[].class);
```

Comment:
good catch.

Reply:
here I have an doub. 

In your PR ->(https://github.com/eclipse/ConfigJSR/pull/84/files#diff-24051286599a5e008d8e368f80fdeb4dL91) 

was as ConfigProvider.getValue and was changed by config.getValue (). I understand that in this section Config was injected or even the ConfigProvider was used.

In case the SPEC must be conformed as it is displayed in the javadoc or in this way it validates anyway?

thanks